scripts/compose.py

```diff
@@ -1,26 +1,22 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
that's the right one?

Reply:
to make it works in Python 2.7 we have to edit the `PYTHONPATH`, on Python 3 it is not needed, so because Python 2.7 is in its end of life I suggest to move forward to Python 3 -> https://pythonclock.org/

Reply:
This needs to be documented in the `README` though, ideally with some instructions how to get Python 3 on OSX (not sure if `compose.py` even works on Windows)

Reply:
it is true python3 is not in Mac OS X by default, we will have to live with python 2.7 for a time

Reply:
Apple is about to deprecate the use of the included Python in OS X 10.15 in October, and will remove it completely in 10.16. So we need to document how to install Python anyway, might as well do it now with Python 3

https://www.macobserver.com/analysis/macos-catalina-deprecates-unix-scripting-languages/

Reply:
I've made it works on Python 2.7